src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject('First promise was rejected');
+  }, 3000);
+
+  document.body.addEventListener(
+    'click',
+    (e) => {
+      resolve('First promise was resolved');
+      clearTimeout(timeoutId);
+    },
+    { once: true },
+  );
+})
```

Comment:
The first Promise is rejected after 3 seconds regardless of whether a click event has occurred or not. You might want to consider clearing the timeout once the 'click' event is fired to avoid this unnecessary rejection.